File: Voronezh/Bars.GkhGji.Regions.Voronezh/libs/B4/model/InterdepartmentalRequestsDTO.js
Ext.define('B4.model.InterdepartmentalRequestsDTO', {
    extend: 'B4.base.Model',
    idProperty: 'Id',

    proxy: {
        type: 'b4proxy',
        controllerName: 'DepartmentlRequestsDTO',
        listAction: 'GetListInterdepartmentalRequests'
    },
    fields: [
        { name: 'Id' },
        { name: 'Date'},
        { name: 'Number'},
        { name: 'Department'},
        { name: 'Answer' },
        { name: 'NameOfInterdepartmentalDepartment' },        
        { name: 'Inspector' },
        { name: 'RequestState' },
        { name: 'FrontControllerName' },
        { name: 'FrontModelName' }
    ]
});

File: Voronezh/Bars.GkhGji.Regions.Voronezh/Entities/SMEV/Helper/InterdepartmentalRequestsDTO.cs
using Bars.Gkh.Entities;
using Bars.GkhGji.Regions.BaseChelyabinsk.Enums;
using System;
using Bars.GkhGji.Regions.BaseChelyabinsk.Enums.SMEV;

namespace Bars.GkhGji.Regions.Voronezh.Entities.SMEV.Helper
{
    public class InterdepartmentalRequestsDTO
    {
        /// <summary>
        /// Дата запроса
        /// </summary>
        public long Id { get; set; }

        /// <summary>
        /// Дата запроса
        /// </summary>
        public DateTime Date { get; set; }

        /// <summary>
        /// Номер запроса
        /// </summary>
        public string Number { get; set; }

        /// <summary>
        /// Ведомость
        /// </summary>
        public string Department { get; set; }

        /// <summary>
        /// ответ
        /// </summary>
        public string Answer { get; set; }

        /// <summary>
        /// Инспектор
        /// </summary>
        public string Inspector { get; set; }

        /// <summary>
        /// Статус запроса
        /// </summary>
        public RequestState RequestState { get; set; }


        /// <summary>
        /// Наименование межведомственного департамента
        /// </summary>
        public NameOfInterdepartmentalDepartment NameOfInterdepartmentalDepartment { get; set; }

        public string FrontControllerName { get; set; }

        public string FrontModelName { get; set; }
    }
}
